Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:alpine
+
+WORKDIR /openmct
+
+COPY . ./
+
+RUN apk update && apk upgrade \
+  && apk add --no-cache git \
+  && npm install \
```

Comment:
Running all of that as a single layer will not give it much opportunity for caching. If anything I'd move `apk` calls as single command above inital COPY. Also it would be more efficient to copy `package.json` alone first and run `npm install` before copying source files and running build. This way you would benefit from cached layers on consecutive builds.

Reply:
I liked 🤩. Thanks for the feedback!